src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheckTest.java

```diff
@@ -167,7 +167,29 @@ public void testInlineDefaultConfiguration() throws Exception {
         };
 
         verifyWithInlineConfigParser(
-                getPath("InputSummaryJavadocInlineDefault.java"), expected);
+            getPath("InputSummaryJavadocInlineDefault.java"), expected);
```

Comment:
please revert this line update, it isnot related and not required.

Reply:
done